File: Solved/The Departed/The Departed.py
# The Departed (2006)
# ID : 132251
# https://quera.org/problemset/132251/
# https://b2n.ir/k77368

results = []

for i in range(1, 6):
    line = input()

    if "FBI" in line:
        results.append(str(i))

if len(results) == 0:
    print("HE GOT AWAY!")

else:
    output = []

    for i in results:
        output.append(i)

    print(" ".join(output))

File: Solved/The Departed/TheDeparted.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

/*
Question title = The Departed (2006)
https://quera.org/problemset/132251/
 */

public class TheDeparted {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        List<String> results = new ArrayList<>();

        for (int i = 1; i <= 5; i++) {
            String line = sc.nextLine();
            if (line.contains("FBI"))
                results.add(i + "");
        }

        if (results.size() == 0) {
            System.out.println("HE GOT AWAY!");
        } else {
            StringBuilder output = new StringBuilder();
            for (String i : results) {
                output.append(i).append(" ");
            }
            output.deleteCharAt(output.length() - 1);

            System.out.println(output.toString());
        }
    }
}